crates/cxx-qt-lib/src/core/init.cpp

```diff
@@ -11,163 +11,102 @@
-static const int register_QVector_f64 =
   qRegisterMetaType<::QVector_f64>("QVector_f64");
-static const int register_QVector_i8 =
   qRegisterMetaType<::QVector_i8>("QVector_i8");
-static const int register_QVector_i16 =
   qRegisterMetaType<::QVector_i16>("QVector_i16");
-static const int register_QVector_i32 =
   qRegisterMetaType<::QVector_i32>("QVector_i32");
-static const int register_QVector_i64 =
   qRegisterMetaType<::QVector_i64>("QVector_i64");
-static const int register_QVector_QByteArray =
   qRegisterMetaType<::QVector_QByteArray>("QVector_QByteArray");
-static const int register_QVector_QDate =
   qRegisterMetaType<::QVector_QDate>("QVector_QDate");
-static const int register_QVector_QDateTime =
   qRegisterMetaType<::QVector_QDateTime>("QVector_QDateTime");
-static const int register_QVector_QLine =
   qRegisterMetaType<::QVector_QLine>("QVector_QLine");
-static const int register_QVector_QLineF =
   qRegisterMetaType<::QVector_QLineF>("QVector_QLineF");
-static const int register_QVector_QMargins =
   qRegisterMetaType<::QVector_QMargins>("QVector_QMargins");
-static const int register_QVector_QMarginsF =
   qRegisterMetaType<::QVector_QMarginsF>("QVector_QMarginsF");
-static const int register_QVector_QPersistentModelIndex =
   qRegisterMetaType<::QVector_QPersistentModelIndex>(
     "QVector_QPersistentModelIndex");
-static const int register_QVector_QPoint =
   qRegisterMetaType<::QVector_QPoint>("QVector_QPoint");
-static const int register_QVector_QPointF =
   qRegisterMetaType<::QVector_QPointF>("QVector_QPointF");
-static const int register_QVector_QRect =
   qRegisterMetaType<::QVector_QRect>("QVector_QRect");
-static const int register_QVector_QRectF =
   qRegisterMetaType<::QVector_QRectF>("QVector_QRectF");
-static const int register_QVector_QSize =
   qRegisterMetaType<::QVector_QSize>("QVector_QSize");
-static const int register_QVector_QSizeF =
   qRegisterMetaType<::QVector_QSizeF>("QVector_QSizeF");
-static const int register_QVector_QString =
   qRegisterMetaType<::QVector_QString>("QVector_QString");
-static const int register_QVector_QTime =
   qRegisterMetaType<::QVector_QTime>("QVector_QTime");
-static const int register_QVector_QUrl =
   qRegisterMetaType<::QVector_QUrl>("QVector_QUrl");
-// Ensure that QVector<QVariant> (aka QVariantList) is registered
-// otherwise it cannot be used in QML
-static const int register_QVector_QVariant =
+  // Ensure that QVector<QVariant> (aka QVariantList) is registered
+  // otherwise it cannot be used in QML
   qRegisterMetaType<::QVector_QVariant>("QVector_QVariant");
-static const int register_QVector_u8 =
   qRegisterMetaType<::QVector_u8>("QVector_u8");
-static const int register_QVector_u16 =
   qRegisterMetaType<::QVector_u16>("QVector_u16");
-static const int register_QVector_u32 =
   qRegisterMetaType<::QVector_u32>("QVector_u32");
-static const int register_QVector_u64 =
   qRegisterMetaType<::QVector_u64>("QVector_u64");
+
+  return 42;
```

Comment:
why do you return 42 ? Seems to be a magic value that isn't used anywhere ? Can you not use `void` ? If you do need to return something to indicate success why not `bool` or true/false ? If this is a magic value then can we have a static const in cxx-qt or exposed from rust somewhere ?

Reply:
Replaced with `true`.